src/utils/alert.ts

```diff
@@ -131,7 +137,7 @@ export const sendAlerts = async ({
     notifications.map((notification) => {
       switch (notification.type) {
         case 'mailgun': {
-          return sendMailgun(
+          sendMailgun(
```

Comment:
Why remove return? sendMailGun returns a Promise that will be caught by Promise.all.

Reply:
solved